lib/js-api/src/alfrescoApi.ts

```diff
@@ -240,6 +249,19 @@ export class AlfrescoApi implements Emitter, AlfrescoApiType {
         });
     }
 
+    ticketMismatchListeners() {
```

Comment:
missing null/undefined checks, I suggest checking the clients like `this.contentClient?.off(...)` and for other cases

Reply:
I've added null/undefined checks as requested.